src/modm/ui/display/color_graphic_display.hpp

```diff
@@ -0,0 +1,117 @@
+#ifndef MODM_COLOR_GRAPHIC_DISPLAY_HPP
+#define MODM_COLOR_GRAPHIC_DISPLAY_HPP
+
+#include "graphic_display.hpp"
+#include <modm/platform.hpp>
+
+using namespace modm::platform;
+
+namespace modm {
+
+// Backwards compatibility
+	namespace glcd
+	{
+		/// @ingroup modm_ui_display
+		typedef Vector<int16_t, 2> Point;
+
+		// RGB16 (565) Format
+		/// @ingroup modm_ui_display
+		class Color
+		{
+		public:
+			static modm_always_inline Color white()   { return Color(0xffff); };
```

Comment:
Every member function here can be `constexpr` so that it can be used in a compile-time evaluation context, also the non-static ones below. 

```suggestion
			static constexpr Color white()   { return Color(0xffff); };
```

modm_always_inline shouldn't be needed here, the compiler will for sure optimize everything away starting from `-Og`.